Using the ``BaseGeometry`` class:

Import the module and create an instance:
    >>> BaseGeometry = __import__('7-base_geometry').BaseGeometry
    >>> my_instance = BaseGeometry()

Validate int value:
    >>> my_instance.integer_validator("size", 8)

Validate wrong type:
    >>> my_instance.integer_validator("size", "1")
    Traceback (most recent call last):
        ...
    TypeError: size must be an integer

Validate Negative int:
    >>> my_instance.integer_validator("size", -8)
    Traceback (most recent call last):
        ...
    ValueError: size must be greater than 0

Validate 0:
    >>> my_instance.integer_validator("size", 0)
    Traceback (most recent call last):
        ...
    ValueError: size must be greater than 0

Validate list value:
    >>> my_instance.integer_validator("size", [0])
    Traceback (most recent call last):
        ...
    TypeError: size must be an integer

Validate tuple value:
    >>> my_instance.integer_validator("size", (1,))
    Traceback (most recent call last):
        ...
    TypeError: size must be an integer

Validate value of None Type:
    >>> my_instance.integer_validator("size", None)
    Traceback (most recent call last):
        ...
    TypeError: size must be an integer

Validate value of Dict Type:
    >>> my_instance.integer_validator("size", {4:3})
    Traceback (most recent call last):
        ...
    TypeError: size must be an integer

Validate bool:
    >>> my_instance.integer_validator("size", True)
    Traceback (most recent call last):
        ...
    TypeError: size must be an integer

Test Area method:
    >>> my_instance.area()
    Traceback (most recent call last):
        ...
    Exception: area() is not implemented

Test integer_validator():
    >>> my_instance.integer_validator()
    Traceback (most recent call last):
        ...
    TypeError: integer_validator() missing 2 required positional arguments: 'name' and 'value

Test integer_validator with one argument:
    >>> my_instance.integer_validator("size")
    Traceback (most recent call last):
        ...
    Exception: area() is not implemented
